ush/global_cycle.sh

```diff
@@ -141,9 +141,11 @@
 #     DONST         Process NST records when using NST model.  Default is 'no'.
 #     DO_SFCCYCLE   Call sfcsub routine 
 #     DO_LNDINC     Call routine to update snow/soil states with increment files
-#     DO_SOI_INC_GSI    Call routine to update soil states with gsi(gaussian) increment files
-#     DO_SNO_INC_JEDI   Call routine to update snow states with jedi increment files
-#     DO_SOI_INC_JEDI   Call routine to update soil states with jedi increment files
+#     DO_SOI_INC    Call routine to update soil states 
```

Comment:
I would add more detail on how to set each logical.

Reply:
I've updated the comments, and added some logic into the global_driver.sh to set DO_LNDINC to true if either of DO_SNO_INC or DO_SOI_INC it true.